mdpow/equil.py

```diff
@@ -610,7 +611,10 @@ def get_last_checkpoint(self):
         """Returns the checkpoint of the most advanced step in the protocol.
         Relies on md.gro being present from previous simulation, assumes that checkpoint is then present.
         """
-        return self._lastnotempty([self.files[name] for name in self.checkpoints]).replace('.gro','.cpt')
+        if self._lastnotempty([self.files[name] for name in self.checkpoints]) is not None:
```

Comment:
Replace with a try/except, especially because you don't want to evaluate `_lastnotempty` twice.

Reply:
Document under which circumstances None is returned.